src/app/qgsmapcanvasdockwidget.cpp

```diff
@@ -473,20 +496,31 @@ void QgsMapCanvasDockWidget::showLabels( bool show )
   mMapCanvas->setMapSettingsFlags( flags );
 }
 
+void QgsMapCanvasDockWidget::autoZoomToSelection( bool show )
```

Comment:
s/show/autoZoom/ ?

Reply:
I prefer autoZoom here as Show implies just show it but not the zoom action.

Reply:
Same thinking here, therefore the proposal to change the parameter name.

Reply:
Ah right the parameter totally overlooked that.